frontend/src/lib/components/icon/IconAlt.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+  import {onMount} from 'svelte';
+  import {FILL} from '$lib/components/shared/colors';
+  import type {IconProps} from './Icon.type';
+  import {ICONS} from './icons';
+
+  type $$Props = IconProps;
+  export let name: $$Props['name'];
+  export let size: $$Props['size'] = 'md';
+  export let color: $$Props['color'] = 'current';
+
+  // Validate name and split path
+  // We need this part-wise approach because of Vite's dynamic import limitations
+  // https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations
+  let folder: string;
+  let filename: string;
+
+  if (name in ICONS) {
+    [folder, filename] = ICONS[name].split('/');
+  } else {
+    folder = 'custom';
+    filename = 'missing_icon';
+  }
+
+  // Load svg contents
+  let svgElement: SVGElement;
+  // The import happens inside onMount to make sure the SVG element is available
+  onMount(() =>
+    import(`./svg/${folder}/${filename}.ts`).then((svg) => {
+      if (svgElement) {
+        svgElement.innerHTML = svg.default;
+      }
+    })
+  );
+
+  // Create class names
+  let classes = 'vaa-icon inline-block';
+
+  // Predefined sizes
+  switch (size) {
+    case 'sm':
+      classes += ' h-16 w-16';
+      break;
+    case 'lg':
+      classes += ' h-32 w-32';
+      break;
+    default:
+      classes += ' h-24 w-24';
+  }
+
+  if (color != null && color in FILL) {
+    classes += ` ${FILL[color]}`;
+  }
```

Comment:
```suggestion
  if (color != null) {
    classes += ` fill-${color}`;
  }

```

would this work? I tested it locally, and it seems to work. This way, we do not have to manually define the `FILL` colours and we can just rely on the Color types.

Reply:
Unfortunately, it doesn't work. On your local version, did you check that the fill classes aren't still somewhere in the code even commented out and rebuilt the app (see [earlier comment](https://github.com/OpenVAA/voting-advice-application/pull/234#discussion_r1382945006))?

<img width="957" alt="image" src="https://github.com/OpenVAA/voting-advice-application/assets/7520743/14695e92-8fa0-4e2b-8450-facf861a8157">

Reply:
There's one option for these, though, if we want to be able to use the `fill-${color}` format. We could [safelist](https://tailwindcss.com/docs/content-configuration#safelisting-classes) all of the relevant `fill`, `bg`, `text` etc. classes in `tailwind.config.cjs`. If you feel that would be a better option, we can do that as well.

Reply:
Let's change these to use the safelists.